org.civicrm.reqangular/js/src/common/models/instances/contact-instance.js

```diff
@@ -16,6 +19,18 @@ define([
          */
         leaveManagees: function (params) {
           return ContactAPI.leaveManagees(this.id, params);
```

Comment:
Why not? This is not a recent change and I think it is absolutely fine to use API in the instance. It is not allowed to use in components/controllers, but in instance I think it is totally fine.

Reply:
Yes it was an old code I know. It will be great if we can refactor this.
I remember discussing with @AkA84 that we should only use Models inside instances and not apis. I will wait for him to confirm.

Reply:
Decided to leave as is. cc @AkA84